release-notes/1.19.1.md

```diff
@@ -0,0 +1,37 @@
+## Introduction
+
+The Anbox Cloud team is pleased to announce the release of Anbox Cloud 1.19.1.
+
+Please see [component versions](https://anbox-cloud.io/docs/ref/component-versions) for a list of updated components.
+
+## New features & improvements
+
+* The `anbox-cloud-appliance status` command now lists the version of the application that you are running.
+* The following snaps will be held from being automatically refreshed. They are refreshed only when the charms are upgraded.
+    - `ams`
+    - `ams-node-controller`
+    - `anbox-cloud-dashboard`
+    - `anbox-stream-agent`
+    - `anbox-stream-gateway`
+    - `lxd`
+* You can now see when an application or its version was created using the `CreatedAt` field listed when you run `amc application show`.
+* Android security updates for September 2023 (see Android Security Bulletin - September 2023 for more information).
+* The Android WebView has been updated to 116.0.5845.163.
```

Comment:
Should we link to https://chromereleases.googleblog.com/2023/08/chrome-for-android-update_02015057758.html?